index.html

```diff
@@ -1222,15 +1222,30 @@ <h4>
           <p its-locale-filter-list="zh-hans" lang="zh-hans">表格、图片等标题栏的位置</p>
           <p its-locale-filter-list="zh-hant" lang="zh-hant">表格、圖片等標題列的位置</p>
           <ol>
+            <li id="hang-lie">
+              <p its-locale-filter-list="en" lang="en">
+                In Mainland China, the direction of row in a table is called 行 which is horizontal, while the direction of column is called 列 which is vertical. In Taiwan and Hong Kong, the direction of row in a table is called 列 which is horizontal, while the direction of column is called 行 which is vertical. The inline direction (either in horizontal mode or vertical mode) is historically called as 行, with the block direction is called 列. However, these terms are not swapped in modern usage when changing writing directions out of habits. The following subpoints will indicate the incrementing direction in table to prevent confusion.
```

Comment:
How about:
In Mainland China, the direction of horizontal rows in a table is called 行 (xíng), and the direction of vertical columns is called 列 (liè).  In Taiwan and Hong Kong, the direction of horizontal rows in a table is called  列 (liè), and the direction of vertical columns is called 行 (xíng). Inline direction (in either horizontal or vertical mode) is historically called 行 (xíng), and block direction is called 列 (liè). However, in modern usage, these terms are habitually not swapped when changing the writing direction. To help avoid confusion, the following numbered list indicates the incremental direction in tables.

Reply:
Having just read the original issue on which this PR is based (https://github.com/w3c/clreq/issues/369), i think i understand this text better, and i think the current text is confusing at best, and misleading at worst.

The text currently proposed for this PR implies that there is a difference in the use of the terms 行 and 列 base on the location of usage, but in fact there isn't. There is only a difference in the arrangement of the text in the table.  行 can be described as the inline progression direction, using CSS terminology.  列 is simply the block progression direction.

Instead of saying "In Mainland China..."  and "In Taiwan and Hong Kong...", it should instead say "In horizontally-set text..." and "In vertically-set text..." (horizontal/vertical writing mode, if you prefer).

So here is what i think is a better version of the text i suggested above:

> The direction of the rows in a table follows the inline progression direction for the text, called 行 (háng). The direction of the columns follows the block progression direction, called 列 (liè). In horizontally-set text (such as that usually used in Mainland China), the row direction (行) is left-to-right, whereas in vertically-set text (such as that used in Taiwan or Hong Kong), the row direction is top-to-bottom. The column direction (列) is top-to-bottom in horizontal text, but right-to-left in vertically-set text.

Reply:
According to [Bobby’s comment](https://github.com/w3c/clreq/issues/369#issuecomment-896110913) under #369, in TC (esp. in Taiwan) nowadays, a visually vertical line in table is call 行 (háng) or 欄 (lán) while a visually horizontal line is call 列 (liè).

It means that *no matter in vertical mode or horizontal mode*, in Taiwan nowadays, the column (visually vertical) is called 直**行**/直欄 and the row (visually horizontal) is called 橫**列**. The Taiwanese idiom “直行横列” literally means “vertical *háng* and horizontal *liè*”. However in Mainland China nowadays, it is quite the reverse.

In short, Taiwan (TC) translates “column” into “行” and “row” into “列”, while Mainland (SC) translates “column” into “列” and “row” into “行” nowadays, regardless of writing modes.

If my understanding is correct, this PR is exactly to clarify the terminological difference between TC and SC.

Reply:
> The text currently proposed for this PR implies that there is a difference in the use of the terms 行 and 列 base on the location of usage,

Yes, nowaday it is based on location of usage. Historically it should've been not the case, but the terminologies have been stuck per region now. 

I think this sentence:
> Inline direction (in either horizontal or vertical mode) is historically called 行 (háng), and block progression direction is called 列 (liè). However.....

can be restructured to:
> Historically, the inline direction (in either horizontal or vertical mode) is called 行 (háng), and the block progression direction is called 列 (liè). However......